.github/workflows/smoke-checks.yml

```diff
@@ -92,18 +88,13 @@ jobs:
     steps:
     - uses: actions/checkout@v2
     - uses: ./.github/actions/bootstrap
-    - name: "Prepare ${{ env.IOS_SIMULATOR_DEVICE }} with iOS ${{ env.IOS_VERSION }}"
-      uses: ./.github/actions/prepare-ios-simulator
-      with:
-        device: ${{ env.IOS_SIMULATOR_DEVICE }}
-        ios: ${{ env.IOS_VERSION }}
-        xcode: ${{ env.XCODE_VERSION }}
     - name: Run UI Tests (Debug)
-      run: bundle exec fastlane test_ui device:"${{ env.IOS_SIMULATOR_DEVICE_WITH_VERSION }}"
+      run: bundle exec fastlane test_ui device:"iPhone 12 Pro (15.5)"
```

Comment:
Isn't it better to keep the env variables?

Reply:
`Any reason why the snapshots went from 1080 to 360 px width?`
- I removed our custom action to prepare the Simulator (which was legacy to save time on CI) and used the default iPhone 12 Pro (15.5.) available on Github Actions.

`Isn't it better to keep the env variables?`
- Those env variables were used only in this job but were set up on the pipeline level. The current implementation aligns with other jobs (unit & e2e).

Reply:
Although, on the SwiftUI repo it uses 1080px width with the same setup, weird 🤔

Reply:
Sorry, my bad, returned the `IOS_SIMULATOR_DEVICE` env variable.

Reply:
The snapshots are recorded in your machine, not in the CI. So I wonder if this is related to updating the snapshot library or just Xcode 14 🤔 But that would mean that snapshot tests would fail for who ever is still on Xcode 13

Reply:
Nevermind, this is because I bumped `swift-snapshot-testing` lib version. Just reverted it back.

Reply:
Makes sense now 👍 But this is actually interesting, it reduced the git repo size. This is probably an optimization. But lets reduce the scope of this PR for now, and we handle that later 👍